DeleteUserDataActivity/backupAndDelete.ts

```diff
@@ -2,12 +2,11 @@ import * as crypto from "crypto";
 
 import { BlobService } from "azure-storage";
 import { sequenceT } from "fp-ts/lib/Apply";
-import * as A from "fp-ts/lib/Array";
+import * as RA from "fp-ts/lib/ReadonlyArray";
```

Comment:
In other files we use `ROA`. Maybe is better use the same name across all the files of the project.

Reply:
Right, done in [[#IOPID-1450] refactor based on comments, part 1](https://github.com/pagopa/io-functions-admin/pull/231/commits/13289a70c86f945983ec6e17fee2ccff25ad2f77)